### DISCUSSION TITLE
群辉docker可以正常安装运行容器，但无法网页访问0.3.1以上的版本

### DISCUSSION THREAD

**Discussion by quickflash2** [usememos/memos#241] [Q&A]
使用群辉docker安装最新版本，可以正常运行容器，但是网页不能访问，后逐一测试之前的版本，0.3.1的版本可以正常运行及访问。想请教大佬是什么问题

**Comment by boojack** [comment]
应该是更换了用 docker 的启动命令，最新的请参考 readme
